examples/analytics/spark-k8s-operator/spark-samples/pyspark-pi-job-yunikorn.yaml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: catalog-sales-test1
+  namespace: spark-team-a
+spec:
+  batchScheduler: "yunikorn"
+  type: Python
+  sparkVersion: "3.1.1"
+  pythonVersion: "3"
+  mode: cluster
+  image: "public.ecr.aws/r1l5w1y9/spark-operator:3.2.1-hadoop-3.3.1-java-11-scala-2.12-python-3.8-latest"
```

Comment:
What repo is this? If not an official image I recommend adding Dockerfile and relevant steps to build and push the image to ECR repo.

Reply:
Added comment to reference public docker imaged used